### PR TITLE
Merge text on what not to send in response to ACKs

### LINKED ISSUE #4833: Recovery: Initialization of min_rtt when used with Cubic
In the Loss Recovery Pseudocode, `min_rtt` is initialized to 0: https://tools.ietf.org/html/draft-ietf-quic-recovery-34#appendix-A.4

This can result in dividing by zero if one implements a Cubic congestion controller ([RFC8312](https://tools.ietf.org/html/rfc8312)) that incorporates `min_rtt` (as the [Linux TCP Cubic implementation does](https://github.com/torvalds/linux/blob/05a59d79793d482f628a31753c671f2e92178a21/net/ipv4/tcp_cubic.c#L266)). Cubic specifies the [formula for `W_est`](https://tools.ietf.org/html/rfc8312#section-4.2), used in congestion avoidance, as:
```
 W_est(t) = W_max*beta_cubic + [3*(1-beta_cubic)/(1+beta_cubic)] * (t/RTT)
```

An endpoint can enter congestion avoidance without receiving any acks for any ack-eliciting packets (only non-ack eliciting packets). In such a scenario, `min_rtt` would still be 0, due to the requirement from [§5.1](https://tools.ietf.org/html/draft-ietf-quic-recovery-34#section-5.1):

```
An RTT sample MUST NOT be generated on receiving an ACK frame that does not newly acknowledge at least one ack-eliciting packet.
```
 Since `min_rtt` is 0 when `w_est` is calculated, the divide by 0 occurs. This could be addressed by initializing `min_rtt` to `kInitialRtt` instead of 0.

### PR DESCRIPTION
**Pull request by ianswett**
Editorial improvement to fix #4833

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]
Caution: this is starting to get to the point that tracking these in AUTH48 will be more difficult.  This is fine, but it is also unnecessary.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -3742,10 +3742,11 @@ ack-eliciting packet.
...
 An endpoint MUST NOT send a non-ack-eliciting packet in response to a
 non-ack-eliciting packet, even if there are packet gaps that precede the
-received packet. This avoids an infinite feedback loop of acknowledgments,
-which could prevent the connection from ever becoming idle.  Non-ack-eliciting
-packets are eventually acknowledged when the endpoint sends an ACK frame in
-response to other events.
+received packet.  Additionally, a receiver MUST NOT send an ack-eliciting
+frame in all packets that would otherwise be non-ack-eliciting. This avoids
+an infinite feedback loop of acknowledgments, which could prevent the
+connection from ever becoming idle.  Non-ack-eliciting packets are eventually
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:3748]:
```suggestion
an infinite feedback loop of acknowledgments.  Non-ack-eliciting packets are eventually
```

I think that the connection between infinite loop and never being idle is clear enough without saying it.

**Review by martinthomson — COMMENTED** [review]
After having discussed this, I think that I'm opposed to this change.  @janaiyengar points out that the moved sentence more directly links to the paragraph preceding it.

There is a gap here.  We don't say here that you might deliberately include ack-eliciting frames in packets that are otherwise non-ack-eliciting so that you can get an acknowledgment.  The moved text here is about this case, saying that you can't do that every time.  It's missing that important context though.

**Comment by janaiyengar** [comment]
I prefer #4859 -- let's do that one instead.

**Pull request closed without merging**